target/src/README.md

```diff
@@ -0,0 +1,83 @@
+# antlr4ts - TypeScript/JavaScript target for ANTLR 4
```

Comment:
❗️ Rather than copy the complete content from the root and expect it to be maintained, use one of the following approaches:

1. Copy the file from the root to this location during the build, and don't include it in source control.
2. Remove all non-essential content from this file so that it's nothing more than a name, brief description, and a link to the *actual* readme which remains at the root of this repository.